### ISSUE TITLE
Truncated file. Aborting.

### ISSUE THREAD

**Issue by pengxiao78** [chapmanb/bcbio-nextgen#1619]
Hi Brad,
Another different error during alignment. Any solution?
Thanks.

[2016-10-28T02:21Z] ***.edu: Aligning lane 13_2016-10-26_13th-merged with bwa aligner
[2016-10-28T02:21Z] ***.edu: Uncaught exception occurred
Traceback (most recent call last):
  File "/path/to/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 21, in run
    _do_run(cmd, checks, log_stdout)
  File "/path/to/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 95, in _do_run
    raise subprocess.CalledProcessError(exitcode, error_msg)
CalledProcessError: Command 'set -o pipefail; /path/to/bcbio/galaxy/../anaconda/bin/bwa mem  -c 250 -M -t 16  -R '@RG\tID:7948tumor\tPL:illumina\tPU:13_2016-10-26_13th-merged\tSM:7948tumor' -v 1 /path/to/bcbio/genomes/Hsapiens/GRCh37/bwa/GRCh37.fa <(grabix grab /path/to/project/13th/work/align_prep/7948tumor_R1.trimmed.fastq.gz 60000001 80000000) <(grabix grab /path/to/project/13th/work/align_prep/7948tumor_R2.trimmed.fastq.gz 60000001 80000000)  | /path/to/bcbio/galaxy/../anaconda/bin/samblaster --addMateTags -M --splitterFile >(/path/to/bcbio/galaxy/../anaconda/bin/samtools sort -@ 16 -m 682M -T /path/to/project/13th/work/align/7948tumor/split/tx/tmpuctFHj/7948tumor-sort-60000001_80000000-sorttmp-spl -o /path/to/project/13th/work/align/7948tumor/split/tx/tmpxf7cxd/7948tumor-sort-60000001_80000000-sr.bam /dev/stdin) --discordantFile >(/path/to/bcbio/galaxy/../anaconda/bin/samtools sort -@ 16 -m 682M -T /path/to/project/13th/work/align/7948tumor/split/tx/tmpuctFHj/7948tumor-sort-60000001_80000000-sorttmp-disc -o /path/to/project/13th/work/align/7948tumor/split/tx/tmp8etX7r/7948tumor-sort-60000001_80000000-disc.bam /dev/stdin) | /path/to/bcbio/galaxy/../anaconda/bin/samtools view -b -S -u - | /path/to/bcbio/galaxy/../anaconda/bin/sambamba sort -N -t 16 -m 682M --tmpdir /path/to/project/13th/work/align/7948tumor/split/tx/tmpuctFHj/7948tumor-sort-60000001_80000000-sorttmp-full -o /path/to/project/13th/work/align/7948tumor/split/tx/tmpuctFHj/7948tumor-sort-60000001_80000000.bam /dev/stdin
samblaster: Version 0.1.23
samblaster: Inputting from stdin
samblaster: Outputting to stdout
samblaster: Opening /dev/fd/62 for write.
samblaster: Opening /dev/fd/63 for write.
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (113, 171, 317)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 725)
[M::mem_pestat] mean and std.dev: (211.10, 138.78)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 929)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (110, 133, 162)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 266)
[M::mem_pestat] mean and std.dev: (137.55, 39.47)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 318)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (173, 461, 770)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 1964)
[M::mem_pestat] mean and std.dev: (485.79, 399.27)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 2561)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (184, 283, 435)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 937)
[M::mem_pestat] mean and std.dev: (309.64, 160.22)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1188)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
samblaster: Loaded 84 header sequence entries.
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (112, 169, 315)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 721)
[M::mem_pestat] mean and std.dev: (208.48, 129.12)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 924)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (110, 133, 162)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 266)
[M::mem_pestat] mean and std.dev: (137.17, 39.40)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 318)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (243, 451, 1209)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3141)
[M::mem_pestat] mean and std.dev: (598.73, 633.55)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 4107)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (188, 299, 408)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 848)
[M::mem_pestat] mean and std.dev: (307.49, 162.12)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1068)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (115, 183, 319)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 727)
[M::mem_pestat] mean and std.dev: (215.22, 137.97)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 931)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (110, 133, 162)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 266)
[M::mem_pestat] mean and std.dev: (137.18, 39.28)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 318)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (202, 403, 1047)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2737)
[M::mem_pestat] mean and std.dev: (583.04, 558.10)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 3582)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (193, 308, 429)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 901)
[M::mem_pestat] mean and std.dev: (323.87, 172.21)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1137)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[fputs] Broken pipe
[W::sam_read1] parse error at line 126086
[bam_sort_core] truncated file. Aborting.
[W::sam_read1] parse error at line 755730
[bam_sort_core] truncated file. Aborting.
[W::sam_read1] parse error at line 3023301
[main_samview] truncated file.
' returned non-zero exit status 1

**Comment by mchimenti** [comment]
I am getting the exact same error, running variant2 tumor/normal on whole genome.  

Error:  

Traceback (most recent call last):
  File "/Shared/Bioinformatics/data/bcbio/bin/bcbio_nextgen.py", line 230, in <module>
    main(**kwargs)
  File "/Shared/Bioinformatics/data/bcbio/bin/bcbio_nextgen.py", line 43, in main
    run_main(**kwargs)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 43, in run_main
    fc_dir, run_info_yaml)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 87, in _run_toplevel
    for xs in pipeline(config, run_info_yaml, parallel, dirs, samples):
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 127, in variant2pipeline
    samples = run_parallel("process_alignment", samples)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 28, in run_parallel
    return run_multicore(fn, items, config, parallel=parallel)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 86, in run_multicore
    for data in joblib.Parallel(parallel["num_jobs"])(joblib.delayed(fn)(x) for x in items):
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 800, in __call__
    while self.dispatch_one_batch(iterator):
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 658, in dispatch_one_batch
    self._dispatch(tasks)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 566, in _dispatch
    job = ImmediateComputeBatch(batch)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 180, in __init__
    self.results = batch()
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 72, in __call__
    return [func(*args, **kwargs) for func, args, kwargs in self.items]
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/utils.py", line 51, in wrapper
    return apply(f, *args, **kwargs)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multitasks.py", line 72, in process_alignment
    return sample.process_alignment(*args)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/sample.py", line 113, in process_alignment
    data = align_to_sort_bam(fastq1, fastq2, aligner, data)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/alignment.py", line 63, in align_to_sort_bam
    names, align_dir, data)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/alignment.py", line 116, in _align_from_fastq
    out = align_fn(fastq1, fastq2, align_ref, names, align_dir, data)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/ngsalign/bwa.py", line 145, in align_pipe
    names, rg_info, data)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/ngsalign/bwa.py", line 155, in _align_mem
    [do.file_nonempty(tx_out_file), do.file_reasonable_size(tx_out_file, fastq_file)])
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 21, in run
    _do_run(cmd, checks, log_stdout)
  File "/Shared/Bioinformatics/data/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 95, in _do_run
    raise subprocess.CalledProcessError(exitcode, error_msg)
subprocess.CalledProcessError: Command 'set -o pipefail; /Shared/Bioinformatics/data/bcbio/galaxy/../anaconda/bin/bwa mem  -c 250 -M -t 16  -R '@RG\tID:M2\tPL:illumina\tPU:1_2016-10-04_bcbio_cancer_dream\tSM:M2' -v 1 /Shared/Bioinformatics/data/bcbio/genomes/Mmusculus/mm9/bwa/mm9.fa <(grabix grab /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align_prep/M2_R1.fastq.gz 260000001 280000000) <(grabix grab /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align_prep/M2_R2.fastq.gz 260000001 280000000)  | /Shared/Bioinformatics/data/bcbio/galaxy/../anaconda/bin/samblaster --addMateTags -M --splitterFile >(/Shared/Bioinformatics/data/bcbio/galaxy/../anaconda/bin/samtools sort -@ 16 -m 1G -T /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align/M2/split/tx/tmpDdk1V9/M2-sort-260000001_280000000-sorttmp-spl -o /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align/M2/split/tx/tmpTErJ_X/M2-sort-260000001_280000000-sr.bam /dev/stdin) --discordantFile >(/Shared/Bioinformatics/data/bcbio/galaxy/../anaconda/bin/samtools sort -@ 16 -m 1G -T /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align/M2/split/tx/tmpDdk1V9/M2-sort-260000001_280000000-sorttmp-disc -o /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align/M2/split/tx/tmpVuJZWY/M2-sort-260000001_280000000-disc.bam /dev/stdin) | /Shared/Bioinformatics/data/bcbio/galaxy/../anaconda/bin/samtools view -b -S -u - | /Shared/Bioinformatics/data/bcbio/galaxy/../anaconda/bin/sambamba sort -N -t 16 -m 1G --tmpdir /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align/M2/split/tx/tmpDdk1V9/M2-sort-260000001_280000000-sorttmp-full -o /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160929-0025_Hansen_643/mouse_vs/bcbio_var2_mouse-merged/work/align/M2/split/tx/tmpDdk1V9/M2-sort-260000001_280000000.bam /dev/stdin
[M::mem_pestat] mean and std.dev: (1634.48, 1565.98)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8087)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
samblaster: Loaded 35 header sequence entries.
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (89, 847, 2103)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6131)
[M::mem_pestat] mean and std.dev: (1175.40, 1331.76)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8145)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (170, 291, 428)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 944)
[M::mem_pestat] mean and std.dev: (298.24, 143.63)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1202)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (162, 811, 2339)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6693)
[M::mem_pestat] mean and std.dev: (1201.67, 1352.67)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8870)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (559, 1143, 2263)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 5671)
[M::mem_pestat] mean and std.dev: (1391.55, 1220.39)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 7375)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (171, 738, 1657)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 4629)
[M::mem_pestat] mean and std.dev: (962.33, 1013.07)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 6115)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (171, 291, 428)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 942)
[M::mem_pestat] mean and std.dev: (298.29, 143.41)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1199)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (228, 374, 3099)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 8841)
[M::mem_pestat] mean and std.dev: (1848.41, 2636.28)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 12394)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (376, 1040, 2431)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6541)
[M::mem_pestat] mean and std.dev: (1558.60, 1600.16)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8596)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (206, 869, 1867)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 5189)
[M::mem_pestat] mean and std.dev: (1122.15, 1235.17)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 6850)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (170, 291, 429)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 947)
[M::mem_pestat] mean and std.dev: (298.27, 143.88)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1206)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (422, 1079, 1942)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 4982)
[M::mem_pestat] mean and std.dev: (1297.70, 1126.54)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 6502)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RR
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (25, 590, 1537)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 4561)
[M::mem_pestat] mean and std.dev: (841.89, 1012.28)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 6073)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (152, 250, 415)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 941)
[M::mem_pestat] mean and std.dev: (277.80, 144.33)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1204)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (249, 486, 5730)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 16692)
[M::mem_pestat] mean and std.dev: (2657.24, 3433.16)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 22173)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (177, 890, 2029)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 5733)
[M::mem_pestat] mean and std.dev: (1167.82, 1387.43)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 7585)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[fputs] Broken pipe
[W::sam_read1] parse error at line 552902
[bam_sort_core] truncated file. Aborting.
[W::sam_read1] parse error at line 662903
[bam_sort_core] truncated file. Aborting.
[W::sam_read1] parse error at line 5165115
[main_samview] truncated file.
' returned non-zero exit status 1

***************
System yaml: 
galaxy_config: universe_wsgi.ini
resources:
  default:
    cores: 16
    jvm_opts:
    - -Xms750m
    - -Xmx3500m
    memory: 3G
  dexseq:
    memory: 10g
  express:
    memory: 8g
  gatk:
    dir: /Shared/Bioinformatics/data/bcbio/toolplus/gatk/3.6-0-g89b7209
    jvm_opts:
    - -Xms4g
    - -Xmx8g
  macs2:
    memory: 8g
  mutect:
    dir: /Shared/Bioinformatics/data/bcbio/toolplus/mutect/1.1.7
  qualimap:
    memory: 4g
  seqcluster:
    memory: 8g
  snpeff:
    jvm_opts:
    - -Xms1g
    - -Xmx4g
********
Config: 

details:
- algorithm:
    aligner: bwa
    coverage_interval: genome
    indelcaller: false
    mark_duplicates: true
    nomap_split_targets: 100
    platform: illumina
    quality_format: standard
    realign: true
    recalibrate: true
    remove_lcr: true
    merge_bamprep: false
    variantcaller:
    - mutect
  analysis: variant2
  description: H1
  files:
  - /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160927-0051_Hansen_643/merged/H1_R1.fastq.gz
  - /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160927-0051_Hansen_643/merged/H1_R2.fastq.gz
  genome_build: GRCh37
  metadata:
    batch: pat1
    phenotype: tumor
- algorithm:
    aligner: bwa
    coverage_interval: genome
    indelcaller: false
    mark_duplicates: true
    nomap_split_targets: 100
    platform: illumina
    quality_format: standard
    realign: true
    recalibrate: true
    remove_lcr: true
    merge_bamprep: false
    variantcaller:
    - mutect
  analysis: variant2
  description: H3
  files:
  - /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160927-0051_Hansen_643/merged/H3_R1.fastq.gz
  - /Shared/Bioinformatics/data/mchiment/hansen_wgs/20160927-0051_Hansen_643/merged/H3_R2.fastq.gz
  genome_build: GRCh37
  metadata:
    batch: pat1
    phenotype: normal
fc_date: '2016-10-04'
fc_name: bcbio_cancer_dream
upload:
  dir: ../final

**Comment by chapmanb** [comment]
Thanks for the reports and sorry about the issue. We have seen some cases of samblaster segfaults/failures during this alignment and de-duplication step, although the failures are generally intermittent and may be related to system resources. We've identified one reproducible case (although it fails at different points on different machines) and submitted a report upstream (GregoryFaust/samblaster#26).

Practically, I submitted a fix to hopefully resolve this by simplifying the command line for cases where you don't need lumpy structural variant calling and the split/discordant reads that samblaster produces. By avoiding these we have less pipes/resource usage which will hopefully avoid some of the issue.

If you update to the latest development:
```
bcbio_nextgen.py upgrade -u development
```
and re-run, hopefully things will work cleanly for you and proceed past alignment. Thanks much for the report.
